README.md

```diff
@@ -1,82 +1,82 @@
-=Wagn: how pioneers roll
+#Wagn: how pioneers roll
 
-==Basics
+[![Build Status](https://semaphoreapp.com/api/v1/projects/0d61c1f9-5ef0-4c5c-89c6-7664e247e4be/294670/badge.png)](https://semaphoreapp.com/ethan/wagn)
```

Comment:
No man, that's my point.  Anyone who is not a core developer is going to install from the gem; that's what the instructions on the README page tell them to do.  If the master branch is red, it doesn't mean the gem is broken.  (We should never push out a gem branch based on broken code.)

**So putting the badge on this page will often tell users "don't install this now" even when it's perfectly safe to install.**

We need a strong separation between communication to core developers and deck managers.  This is putting a core developer message at the top of the deck manager communication space.